boards/arm/adafruit_trinket_m0/doc/index.rst

```diff
@@ -98,7 +98,12 @@ Programming and Debugging
 *************************
 
 The Trinket M0 ships the BOSSA compatible UF2 bootloader.  The
-bootloader can be entered by quickly tapping the reset button twice.
+bootloader can be entered manually by quickly tapping the reset button
+twice, or optionally via a command over USB.
```

Comment:
This somehow implies that USB device stack and CDC ACM is always present (even if one would read the text below).

Reply:
Reworded.  PTAL.